request.coffee

```diff
@@ -2,17 +2,55 @@ _ = require 'lodash'
 request = require 'request'
 Promise = require 'bluebird'
 PinejsClientCore = require './core'
+BluebirdLRU = require 'bluebird-lru-cache'
 
 request = Promise.promisify(request)
 
+validParams = ['cache']
 module.exports = class PinejsClientRequest extends PinejsClientCore(_, Promise)
+	constructor: (params, backendParams) ->
```

Comment:
Are these backendParams specifically only for the case where you're using the node backend rather than angular?